conda-store-server/conda_store_server/action/generate_lockfile.py

```diff
@@ -61,3 +61,17 @@ def action_solve_lockfile(
 
     with lockfile_filename.open() as f:
         return yaml.safe_load(f)
+
+
+@action.action
+def action_save_lockfile(
+    context,
+    specification: schema.LockfileSpecification,
+):
+    lockfile = specification.dict()["lockfile"]
```

Comment:
Add a comment here explaining that it's necessary to call it this way to serialize lockfile properly, otherwise version will not be in the output.